library/data/hash_map.lean

```diff
@@ -476,10 +476,14 @@ theorem find_iff (m : hash_map α β) (a : α) (b : β a) :
   m.find a = some b ↔ sigma.mk a b ∈ m.entries :=
 m.is_valid.find_aux_iff _ _ _
 
-theorem contains_iff (m : hash_map α β) (a : α) (b : β a) :
+theorem contains_iff (m : hash_map α β) (a : α) :
   m.contains a ↔ a ∈ m.keys :=
 m.is_valid.contains_aux_iff _ _
 
+lemma not_mem_mk_hash_map (hash_fn : α → nat) (nbuckets := 8) (a : α) :
+  a ∉ (mk_hash_map hash_fn nbuckets : hash_map α β) :=
+dec_trivial
+
```

Comment:
Is the default value 8 intended? It is in the middle of two explicit ones. So, it will only fire in partial apps where only the first explicit argument is provided.